### ISSUE TITLE
ubuntu install fails when running under esxi

### ISSUE THREAD

**Issue by nickhilliard** [inex/IXP-Manager#391]
The `apt-get -f dist-upgrade` command in the installer hangs on esxi installs with the following errror in the logs:

```
Setting up open-vm-tools (2:10.0.7-3227872-5ubuntu1~16.04.2) ...

Configuration file '/etc/vmware-tools/tools.conf'
 ==> Modified (by you or by a script) since installation.
 ==> Package distributor has shipped an updated version.
   What would you like to do about it ?  Your options are:
    Y or I  : install the package maintainer's version
    N or O  : keep your currently-installed version
      D     : show the differences between the versions
      Z     : start a shell to examine the situation
 The default action is to keep your current version.
*** tools.conf (Y/I/N/O/D/Z) [default=N] ?
```

Obviously, you can get the install to proceed by pressing <return> at the correct point in time on the install window.  But it's not obvious what's happening unless you're tailing the logs.

**Comment by barryo** [comment]
@nickhilliard - is that from a bare bones unused install of Ubuntu?

**Comment by nickhilliard** [comment]
yip, barebones with only SSH package selected during install.

**Comment by barryo** [comment]
Around line 94 can you try replacing these lines:

```sh
debconf-set-selections <<< "console-setup console-setup/charmap47 select UTF-8"
debconf-set-selections <<< "console-setup console-setup/codeset47 select Lat15"

echo -n "Doing a full system upgrade to ensure latest packages are installed (be patient)... "
log_break && apt-get dist-upgrade -yq &>> /tmp/ixp-manager-install.log
```

with these lines:

```sh
debconf-set-selections <<< "console-setup console-setup/charmap47 select UTF-8"
debconf-set-selections <<< "console-setup console-setup/codeset47 select Lat15"
export DEBIAN_FRONTEND=noninteractive

echo -n "Doing a full system upgrade to ensure latest packages are installed (be patient)... "
log_break && apt-get dist-upgrade -o "Dpkg::Options::=--force-confnew" --force-yes -yq &>> /tmp/ixp-manager-install.log
```

**Comment by nickhilliard** [comment]
this fixes it but ``--force-confnew`` is pretty hardcore and could break things.  ``--force-confold`` would be a safer option.

``--force-yes`` is deprecated.  It's not necessary for this change anyway.

fixed in ab3b96eefe8b4de37c8a13a1cba81dab74dec6d0.
